test/asyncapi_test.go

```diff
@@ -40,6 +40,34 @@ func (s *CLITestSuite) TestAsyncApiExport() {
 			s.Error(nil, "spec generated does not match the template output file")
 		}
 	}
+}
 
+func (s *CLITestSuite) TestAsyncApiImport() {
+	tests := []CLITest{
+		{args: "asyncapi import", exitCode: 1, fixture: "asyncapi/err_no_file.golden"},
```

Comment:
Please look at other tests for an example of how to format your golden files. We use lower-kebab-case and prefix the filename with the command so they're easier to distinguish.
```suggestion
		{args: "asyncapi import", fixture: "asyncapi/import-err-no-file.golden", exitCode: 1},
```